package.json

```diff
@@ -47,6 +47,7 @@
     "graphql-syntax-highlighter-react": "^0.3.5",
     "husky": "^4.2.5",
     "identity-obj-proxy": "^3.0.0",
+    "ignore-loader": "^0.1.2",
```

Comment:
I had to add this file because when I was trying to run the app it was explicitly saying that "ignore-loader" module not found :/